### DISCUSSION TITLE
Get nodal offset for periodic faces

### DISCUSSION THREAD

**Discussion by ambehnam** [neperfepx/neper#772] [General]
Hi, I have created a simple periodic cube using the following commands. I was wondering how I can output the nodal offset vectors for periodic faces. For example a text file with this template: Node1_face1 Node1_face2 0.0 1.0 0.0
```
neper -T -dim 3 -domain 'cube(1,1,1)' -periodicity all -n from_morpho -id 1 -morpho 'cube(2,2,2)' -format tess -o periodic_cube
neper -M periodic_cube.tess -dim 3 -rcl 0.23 -clratio 2:2:2 -elttype hex -order 1
neper -V periodic_cube.tess,periodic_cube.msh -dataelsetcol id -print periodic_cube
```
Thanks,
Amir

**Comment by rquey** [comment]
Use `-format per`.

**Reply by ambehnam** [reply]
I tried -format per in the first line, but it doesn't generate a .per file, and it doesn't add the periodicity data to the .msh file. I have version 4.1.3-11 installed.
